### PR TITLE
Update R.14 contents

### PR DESCRIPTION
**Pull request by gjxo12**
Added content and examples for guideline R.14: Avoid [] parameters, prefer span.

### CODE REVIEW

File: sections/Resource.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 * [R.11: 명시적인 `new`와 `delete` 호출을 지양하라](#Rr-newdelete)
 * [R.12: 명시적인 할당의 결과는 즉시 관리 개체에 전달하라](#Rr-immediate-alloc)
 * [R.13: 하나의 표현식 구문에서 명시적 자원 할당은 최대 한번만 수행하라](#Rr-single-alloc)
-* [R.14: ??? 배열 vs. 포인터 매개변수](#Rr-ap)
+* [R.14: [] 파라미터는 피하고 span을 사용하라](#Rr-ap)
```
Comment (utilForever) [inline sections/Resource.md]:
```suggestion
* [R.14: `[]` 매개변수 대신 `span`을 사용하라](#Rr-ap)
```

File: sections/Resource.md [hunk 2]
```diff
@@ -503,29 +503,30 @@ C++ 표준뿐만 아니라 대부분의 경우 참조는 소유를 하지 않는
 
 * 표현식 내에서 여러번의 명시적 자원 할당이 있다면 경고하라(문제: 직접적인 자원 할당을 얼마나 많이 인지할 수 있을 것인가?)
 
-### <a name="Rr-ap"></a>R.14: ??? 배열 vs. 포인터 매개변수
+### <a name="Rr-ap"></a>R.14: [] 파라미터는 피하고 span을 사용하라
```
Comment (utilForever) [inline sections/Resource.md]:
```suggestion
### <a name="Rr-ap"></a>R.14: `[]` 매개변수 대신 `span`을 사용하라
```

File: sections/Resource.md [hunk 3]
```diff
@@ -503,29 +503,30 @@ C++ 표준뿐만 아니라 대부분의 경우 참조는 소유를 하지 않는
...
+                            // (a pointer should point to a single object, do not subscript)
 
-길이 정보를 유지하기 위해 `span`을 사용하라
+    void f(gsl::span<int>); // good, recommended
+```
 
 ##### Enforcement
 
-매개변수에 `[]`가 사용되면 지적하라.
+`[]` 파라미터 대신 `span`을 사용하라.
```
Comment (utilForever) [inline sections/Resource.md]:
```suggestion
`[]` 매개변수 대신 `span`을 사용하라.
```

**Review by utilForever — CHANGES_REQUESTED** [review]
Can you check my comments? Thanks.

**Comment by gjxo12** [comment]
I checked that commit :) 
Thanks for pointing out what I missed :)

**Review by utilForever — APPROVED** [review]
LGTM now. Thanks for your work! 🏆

**Review by luncliff — APPROVED** [review]
Appreciate your contribution 👍 👍

**Pull request merged**